Fixes charcoal.

PJB3005 opened the pull request:
Charcoal now gets used up slower, and gets instantly cleared from the system when all other reagents are gone. Vomiting no longer deletes charcoal. The rate of reagent removal is increased by a factor of 5.

🆑 
- tweak: charcoal works faster, longer, and doesn't remove itself upon vomiting anymore.

kilozombie commented:
Should have:
1. Made an issue asking if water+potass should be removed.
2. Discovered that charcoal is the more interesting, intended route, but is horrible.
3. Fixed charcoal and removed the water+potass solution.
4. Avoided controversy, avoided bullshit borderline-self-merge, avoided shitstorm

I guess controversy must get you off. This PR works.

ShiftyRail commented:
Feel free to close #17159 as you've been first and mine is much less complete

PJB3005 commented:
Yeah but this was more fun.

On 21 Jan 2018 01:57, "kilozombie" <notifications@github.com> wrote:

> Should have:
>
>    1. Made an issue asking if water+potass should be removed.
>    2. Discovered that charcoal is the more interesting, intended route,
>    but is horrible.
>    3. Fixed charcoal and removed the water+potass solution.
>    4. Avoided controversy, avoided bullshit borderline-self-merge,
>    avoided shitstorm
>
> I guess controversy must get you off. This PR works.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/vgstation-coders/vgstation13/pull/17158#issuecomment-359214642>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHu1wwKmMXHYIzYav1yojNSJ39xd2P4oks5tMotcgaJpZM4RlpfU>
> .
>

ShiftyRail commented:
>fun

Childish, unnecessary, shameful and a bad rewind of something that _already happened_ and was a _fucking shitshow for all parties involved_ would be better epithets

DrCelt commented:
@ShiftyRail this is the only place that @PJB3005 has any semblance of control as he's lost control of his life please do not take this from him

ShiftyRail commented:
@DrCelt don't involve me in your personal differences, please. Thanks

SarahJohnson489 commented:
>fun

Fucking kill yourself. You aren't upholding the responsibilities of your position and you're being a brat.

aceedex commented:
I'd rather the waterpotass be changed back desu

PJB3005 commented:
Not happening

DrCelt commented:
@aceedex sorry lad but the collabs (who don't play the game) have made their decision (this affects people that play the game)

MechaMusume commented:
Maybe you should put big changes to stuff as a vote for the people that actually play to weigh in on instead of trying to force more of your junk on people?

Boxe commented:
🍆

PJB3005 commented:
Votes that are never biased by powergamers and people unwilling to do
discussion.

On 21 Jan 2018 02:25, "MechaMusume" <notifications@github.com> wrote:

> Maybe you should put big changes to stuff as a vote for the people that
> actually play to weigh in on instead of trying to force more of your junk
> on people?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/vgstation-coders/vgstation13/pull/17158#issuecomment-359216057>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHu1w4sKmf0ETz3S_84Xn-iEjWVpjWyzks5tMpIGgaJpZM4RlpfU>
> .
>

DrCelt commented:
yeah I hate it when POWERGAMERS who PLAY THE GAME try to PLAY THE GAME the way I DON'T LIKE

when was the last time you played medbay PJB?

MechaMusume commented:
He'll just dodge the question because himself and nexis never play.

SarahJohnson489 commented:
So @PJB3005 , how about this, instead of merging it in two hours, ya leave something that has gotten along fine for years and get some feedback on your proposed improvement. Ya find out, unbeknownst to you two who no longer play, your improvement creates an issue. Questions are asked to further identify what it is, why anyone should care, and what can be done about it. Debate is had, proposals forwarded and scrutinized, it's resolved and moves forward because gee willickers it's a trivial step down from explosion cheesing and all the problems that come from removing explosion clearing are dealt with. No drama spike from not following the rules for the xth time while @d3athrow is mia and not holding you accountable. No irritation over an underaged child who not only fails to grasp his attitude problem but values it. Y'know, the way things are supposed to be done, the path reached after countless generations of collaborative decision-making. The way you hold others to with your flags you hold partially for this purpose but apparently neither yourself nor nexis.

Wouldn't that just be crazy???

SarahJohnson489 commented:
Still waiting on a response @PJB3005 
All of this was completely and trivially avoidable. What's up.

9600bauds commented:
Please don't self-merge balance PRs, especially in less than a couple hours after making them.

9600bauds commented:
Also you didn't add a changelog, why did you remove that tag?

PJB3005 commented:
Hey if you want me to tell N3X to merge it that's fine too.

On 21 Jan 2018 03:07, "9600bauds" <notifications@github.com> wrote:

> Please don't self-merge balance PRs, especially in less than a couple
> hours after making them.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/vgstation-coders/vgstation13/pull/17158#issuecomment-359217996>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHu1w6FS2sdmdHgTPL8ZS6eZbEazHnXgks5tMpu9gaJpZM4RlpfU>
> .
>
